# GoalPosition message

Header header

int32[] id_array
int32[] position_array
int32[] speed_array